src/config/packets.h

```diff
@@ -9,16 +9,12 @@
  **/
 
 #ifndef PACKETVER
+	///Do not edit this line. To set your client version, please do this instead.
+	///In Linux: run the following command: ./configure --enable-packetver=YYYYMMDD
+	///In Windows: add this line in your src\custom\defines_pre.h file: #define PACKETVER YYYYMMDD
```

Comment:
The `defines_pre.h` can also be used for Linux, not just Windows.

The documentation is good, we just don't want it to be misleading that it's only applicable to Windows builds.

Reply:
What about...

> ///In Windows: add this line in your src\custom\defines_pre.h file: #define PACKETVER YYYYMMDD
> ///In Linux: The same as above, or run the following command: ./configure --enable-packetver=YYYYMMDD